.deepsource.toml

```diff
@@ -0,0 +1,12 @@
+version = 1
+
+test_patterns = ["tests/**"]
+
+exclude_patterns = ["docker/**"]
```

Comment:
What about the docs folder?

Reply:
That can be excluded too, but I saw the `conf.py` file in there, so I didn't add the docs folder here.